src/platform_storage_api/admission_controller/volume_resolver.py

```diff
@@ -85,15 +122,19 @@ async def __aenter__(self) -> "KubeVolumeResolver":
         internal state updater, so we can resolve the real mount paths.
         """
         logger.info("initializing volume resolver")
-        namespace_url = self._kube.generate_namespace_url()
         pod_name = socket.gethostname()
 
         try:
-            pod = await self._kube.get(f"{namespace_url}/pods/{pod_name}")
+            pod = await self._kube.get_pod(pod_name)
         except Exception as e:
             raise VolumeResolverError() from e
 
         await self._refresh_internal_state(pod=pod)
+
+        if not self._local_fs_prefix_to_kube_volume:
```

Comment:
let's not raise error. If there are no storages then ok. May be in future storage will be optional